src/components/input/InputRoot.tsx

```diff
@@ -37,6 +37,7 @@ const InputRoot = function InputRoot({
 
   ...htmlAttributes
 }: InputRootProps) {
+  /* istanbul ignore next */
```

Comment:
Ignoring for now, but we need to discuss how to test components which props only apply classes dynamically, without coupling tests with tailwind.